### PR TITLE
VD for news

### PR DESCRIPTION
**Pull request by kobsonaut**

### CODE REVIEW

File: TOZ_iOS/GalleryEntity.swift [hunk 1]
```diff
@@ -0,0 +1,16 @@
+//
+//  GalleryEntity.swift
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryEntity.swift]: What is it? I was removed long time ago ;)

File: TOZ_iOS/NewsResponseMapper.swift [hunk 2]
```diff
@@ -21,7 +21,8 @@ final class NewsResponseMapper: ArrayResponseMapper<NewsItem>, ResponseMapperPro
             }
             var photoURL: URL? = nil
             if let photoUrl = photoUrl {
-                photoURL = BackendConfiguration.shared.baseURL.appendingPathComponent(photoUrl)
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/NewsResponseMapper.swift]: Why?

File: TOZ_iOS/BackendService.swift [hunk 3]
```diff
@@ -18,8 +18,8 @@ Tries to serialize the response data to json.
 class BackendService {
 
     private let conf: BackendConfiguration
-    private let service = NetworkService() // commented in order to run NetworkServiceMock
-//    private let service = NetworkServiceMock() // test of NetworkService
+//    private let service = NetworkService() // commented in order to run NetworkServiceMock
+    private let service = NetworkServiceMock() // test of NetworkService
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/BackendService.swift]: You shouldn't commit this. It's only for development purposes.

Comment (kobsonaut) [inline TOZ_iOS/BackendService.swift]: Yes, I did it by mistake when I was checking the view.

File: TOZ_iOS/NewsResponseMapper.swift (hunk 2 above)
Comment (kobsonaut) [inline TOZ_iOS/NewsResponseMapper.swift]: Do you mean it should be: photoURL = URL(string: photoUrl)?

File: TOZ_iOS/GalleryEntity.swift (hunk 1 above)
Comment (kobsonaut) [inline TOZ_iOS/GalleryEntity.swift]: Wasn't it updated?

**Pull request closed without merging**